src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -860,6 +860,20 @@ void GeneralPage::initLayout(GeneralPageView &layout)
         s.attachExtensionToAnyProcess, false,
         "Attempt to force the Chatterino Browser Extension to work in certain "
         "browsers that do not work automatically.\ne.g. Librewolf");
+    layout.addSubtitle("Additional extension IDs");
+    layout
+        .addDescription("A semicolon separated list of Chrome or Firefox "
```

Comment:
I feel like this reads better, but feel free to skip this if you're an english professional
```suggestion
        .addDescription("A semicolon-separated list of Chrome or Firefox "
```